src/pages/organizations.tsx

```diff
@@ -28,9 +30,14 @@ const Organizations: FC<OrganizationsProps> = () => {
     }
 
     getAllOrganizations(accessToken).then((orgs) => {
+      // When only one organization, redirect to that organization
+      if (orgs.length === 1) {
```

Comment:
Just to confirm since I'm not great at reasoning about when `useEffect` runs (I kind of miss `componenetDidMount` 🤣 🙃 ), this effectively only runs on first login, correct?

Reply:
This runs any time the page is loaded 😄